Add note about nested arrays (#4608)
segmentio/segment-docs: src/connections/destinations/catalog/amplitude/index.md

### I Don't See My Data In Amplitude

> If you aren't seeing your data arrive in Amplitude, take a look at our Analytics.js [guide to validating data being transmitted](/docs/connections/sources/catalog/libraries/website/javascript/troubleshooting#is-data-being-transmitted-to-your-third-party-destinations) to your third-party destination. Please also note that Amplitude doesn't support fields with a value of an array with nested arrays.

Reviewer: ```suggestion
If you aren't seeing your data arrive in Amplitude, see the Analytics.js [guide to validating data being transmitted](/docs/connections/sources/catalog/libraries/website/javascript/troubleshooting#is-data-being-transmitted-to-your-third-party-destinations) to your third-party destination. Amplitude doesn't support fields with a value of an array with nested arrays.
```
